node/src/reactor/main_reactor/tests/binary_port.rs

```diff
@@ -196,15 +202,20 @@ async fn binary_port_component() {
 
     let (
         client,
-        (finish_cranking, mut rng, network_chainspec_raw_bytes, highest_block, secret_key),
+        (
+            finish_cranking,
+            TestData {
+                mut rng,
+                chainspec_raw_bytes: network_chainspec_raw_bytes,
+                highest_block,
+                secret_signing_key,
+            },
+        ),
     ) = setup().await;
 
     let test_cases = &[
-        block_hash_2_height(),
-        highest_complete_block(),
-        completed_blocks_contain_true(),
-        completed_blocks_contain_false(),
-        transaction_hash_2_block_hash_and_height(&mut rng),
+        block_header_info(*highest_block.block().hash()),
+        signed_block_info(*highest_block.block().hash()),
```

Comment:
These 3 were deleted:
- BlockHash2Height
- HighestCompleteBlock
- CompletedBlocksContain
- TransactionHash2BlockHashAndHeight

The reason is that during review we noticed they become redundant once we add the new `InformationRequest`, which support retrieving complete block and transaction information with the `BlockIdentifier` enum which already supports both height and hash.